### PR TITLE
docs(tuto): add meilisearch tuto

### PR DESCRIPTION
**Pull request by nerda-codes**

### CODE REVIEW

File: tutorials/deploy-meilisearch-instance/index.mdx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    </Message>
+6. Use the following command to check that Meilisearch is running.
+
+    ```
+    curl -X GET 'http://localhost:7700/version' -H 'Authorization: Bearer '$MEILI_MASTER_KEY
+    ```
+
+    You should see an output similar to the following:
+    ```json
+    {"commitSha":"0df84bbba7db46680e83843dd6454257161a282f","commitDate":"2024-06-27T12:47root@scwrororororrrrrrrrrrrrrrrrorrrorrrrrrrrrrrrroot@scw-lucid-satoshi:~#
```
Comment (fpagny) [inline tutorials/deploy-meilisearch-instance/index.mdx:77]: This line feels like there is a copy paste issue.
The format should be json only, like {"commitSha":"0df84bbba7db46680e83843dd6454257161a282f","commitDate":"2024-06-27T12", "pkgVersion":aaaaaa}

**Review by fpagny — CHANGES_REQUESTED** [review]

File: tutorials/deploy-meilisearch-instance/index.mdx (hunk 1 above)
Comment (nerda-codes) [inline tutorials/deploy-meilisearch-instance/index.mdx:77]:
```suggestion
    {"commitSha":"0df84bbba7db46680e83843dd6454257161a282f","commitDate":"2024-06-27T12", "pkgVersion":aaaaaa}
```

**Review by fpagny — APPROVED** [review]

File: tutorials/deploy-meilisearch-instance/index.mdx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+## Connecting to your Instance and starting Meilisearch
+
+1. Open a terminal and connect to your Instance using [SSH](/compute/instances/how-to/connect-to-instance/). Replace `<INSTANCE_IP>` with your Instance's IP (Instances > Overview > Flexible IP > the first IP address that displays in the list).
+
+    ```
+    ssh root@<INSTANCE_IP>
+    ```
+
+    <Message type="tip">
+     If you are using an Instance with a Docker InstantApp, run `docker -v` to make sure that the Docker engine is already installed. The current docker engine version should display.
```
Comment (RoRoJ) [inline tutorials/deploy-meilisearch-instance/index.mdx:37]:
```suggestion
     If you are using an Instance with a Docker InstantApp, run `docker -v` to make sure that the Docker engine is already installed. The current Docker engine version should display.
```

File: tutorials/deploy-meilisearch-instance/index.mdx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+### Going further
+
+- Learn more about [Meilisearch APIs and features](https://www.meilisearch.com/docs/reference/api/overview#parameters)
+
+- **Secure** your deployment:
+  - Encrypt traffic in transit to Meilisearch by [setting up SSL](https://www.meilisearch.com/docs/learn/configuration/instance_options#ssl-options)
+  - Move your Instance inside a [Private Network](/compute/instances/how-to/use-private-networks/) if you do not want it to be exposed publicly on the internet
+  - Configure regular data backups using [Meilisearch Snapshots](https://www.meilisearch.com/docs/learn/advanced/snapshots) or [Instances snapshots](/compute/instances/how-to/create-a-snapshot/)
+
+- **Fine-tune deployment configuration** such as [Instances or disk size](/compute/instances/reference-content/choosing-instance-type/), from the [Scaleway console](https://console.scaleway.com/instance/servers), [the Scaleway API](https://www.scaleway.com/en/developers/api/instance/), [CLI](/compute/instances/api-cli/cli-cheatsheet/) or [Terraform](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/instance_server).
```
Comment (RoRoJ) [inline tutorials/deploy-meilisearch-instance/index.mdx:165]:
```suggestion
- **Fine-tune deployment configuration** such as [Instance type or disk size](/compute/instances/reference-content/choosing-instance-type/), from the [Scaleway console](https://console.scaleway.com/instance/servers), [the Scaleway API](https://www.scaleway.com/en/developers/api/instance/), [CLI](/compute/instances/api-cli/cli-cheatsheet/) or [Terraform](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/instance_server).
```

File: tutorials/deploy-meilisearch-instance/index.mdx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+## Troubleshooting
+
+If you encounter any issues, check that you meet all the [requirements listed at the beginning of this page](#before-you-start). Also, verify that your Instance accepts connections from your computer. To do so, you can configure [security groups](/compute/instances/how-to/use-security-groups/) or a [Public Gateway](/network/public-gateways/how-to/configure-a-public-gateway/) to the [Private Network](/compute/instances/how-to/use-private-networks/) your Instance belongs to.
```
Comment (RoRoJ) [inline tutorials/deploy-meilisearch-instance/index.mdx:169]:
```suggestion
If you encounter any issues, check that you meet all the [requirements listed at the beginning of this page](#before-you-start). Also, verify that your Instance accepts connections from your computer. To do so, you can configure [security groups](/compute/instances/how-to/use-security-groups/) or a [Public Gateway](/network/public-gateways/how-to/configure-a-public-gateway/) on the [Private Network](/compute/instances/how-to/use-private-networks/) your Instance belongs to.
```

File: tutorials/deploy-meilisearch-instance/index.mdx [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+  paragraph: Deploy Meilisearch on Scaleway Instances with this step-by-step guide to set up and run the Meilisearch search engine efficiently. Perfect for fast, relevant searches.
+tags: meilisearch search full-text instance
+categories:
+  - instances
+dates:
+  validation: 2024-07-25
+  posted: 2024-07-25
+---
+
+[Meilisearch](https://www.meilisearch.com/) is an open-source search engine that provides instant and relevant search results from large collections of data. Meilisearch is particularly useful for applications that require fast and efficient search capabilities, such as e-commerce sites, documentation search, and any platform where users need to quickly find relevant information from a large dataset.
```
Comment (bene2k1) [inline tutorials/deploy-meilisearch-instance/index.mdx:16]:
```suggestion
Meilisearch is an open-source search engine that provides instant and relevant search results from large collections of data. Meilisearch is particularly useful for applications that require fast and efficient search capabilities, such as e-commerce sites, documentation search, and any platform where users need to quickly find relevant information from a large dataset.
```

removed first link for seo

File: tutorials/deploy-meilisearch-instance/index.mdx [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+  - Configure regular data backups using [Meilisearch Snapshots](https://www.meilisearch.com/docs/learn/advanced/snapshots) or [Instances snapshots](/compute/instances/how-to/create-a-snapshot/)
+
+- **Fine-tune deployment configuration** such as [Instances or disk size](/compute/instances/reference-content/choosing-instance-type/), from the [Scaleway console](https://console.scaleway.com/instance/servers), [the Scaleway API](https://www.scaleway.com/en/developers/api/instance/), [CLI](/compute/instances/api-cli/cli-cheatsheet/) or [Terraform](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs/resources/instance_server).
+
+## Troubleshooting
+
+If you encounter any issues, check that you meet all the [requirements listed at the beginning of this page](#before-you-start). Also, verify that your Instance accepts connections from your computer. To do so, you can configure [security groups](/compute/instances/how-to/use-security-groups/) or a [Public Gateway](/network/public-gateways/how-to/configure-a-public-gateway/) to the [Private Network](/compute/instances/how-to/use-private-networks/) your Instance belongs to.
+
+
+
```
Comment (bene2k1) [inline tutorials/deploy-meilisearch-instance/index.mdx:172]:
```suggestion
```

**Review by bene2k1 — APPROVED** [review]

**Review by RoRoJ — APPROVED** [review]

File: tutorials/deploy-meilisearch-instance/index.mdx [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+  posted: 2024-07-25
+---
+
+Meilisearch is an open-source search engine that provides instant and relevant search results from large collections of data. [Meilisearch](https://www.meilisearch.com/) is particularly useful for applications that require fast and efficient search capabilities, such as e-commerce sites, documentation search, and any platform where users need to quickly find relevant information from a large dataset.
+
+This tutorial shows you how to deploy a Meilisearch search engine on a [Scaleway Instance](/compute/instances/).
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
```
Comment (nerda-codes) [inline tutorials/deploy-meilisearch-instance/index.mdx:22]:
```suggestion
- A Scaleway account logged into the [console](https://console.scaleway.com/)
```

**Pull request merged**